patches/minecraft/net/minecraft/inventory/ContainerPlayer.java.patch

```diff
@@ -1,11 +0,0 @@
---- ../src-base/minecraft/net/minecraft/inventory/ContainerPlayer.java
-+++ ../src-work/minecraft/net/minecraft/inventory/ContainerPlayer.java
-@@ -43,7 +43,7 @@
-                 }
-                 public boolean func_75214_a(ItemStack p_75214_1_)
-                 {
--                    return entityequipmentslot == EntityLiving.func_184640_d(p_75214_1_);
-+                    return p_75214_1_.func_77973_b().isValidArmor(p_75214_1_, entityequipmentslot, field_82862_h);
```

Comment:
You deprecate this fucntion and do nothing with it to make the new system return the proper value for existing mods with this override.
This is NOT deprecation this is removal. Fix that